config/canonical.json

```diff
@@ -31382,6 +31382,8 @@
     "count": 152,
     "tags": {
       "brand": "Tesco Metro",
+      "brand:wikidata": "Q487494",
+      "brand:wikipedia": "en:Tesco",
```

Comment:
...and maybe it is better to remove the `brand:wikipedia` tag because the actual brand is Tesco Metro and not Tesco...

Reply:
okey dokey